benchmark/realnest/09_array_agg.benchmark

```diff
@@ -0,0 +1,26 @@
+# name: benchmark/realnest/09_array_agg.benchmark
+# description: Aggregate nested structs
+# group: [realnest]
+
+name aggregate_nested_structs
+group real_nest
+
+require json
+
+require httpfs
+
+load benchmark/realnest/load.sql
+
+run
```

Comment:
Is it intentional to have two `SELECT` in the same benchmark here?

Reply:
Oh, sorry, it was accidentally. I'll make sure to remove the extra `SELECT`